site/content/docs/cli/constructs-reference.md

```diff
@@ -59,8 +60,9 @@ settings apply to your Checks. Takes all [Check properties](#check)
 dedicated docs on checkMatch and testMatch](/docs/cli/using-check-test-match/)
 
 - `cli`: Defaults for CLI commands.
-  - `runLocation`: The default run location when running `npx checkly test`.
-  - `privateRunLocation`: The default private run location when running `npx checkly test`.
+  - `runLocation`: The default run location when running `npx checkly test` and `npx checkly trigger`.
+  - `privateRunLocation`: The default private run location when running `npx checkly test` and `npx checkly trigger`.
+  - `retries`: The number of times to retry a failing check run when running `npx checkly test` and `npx checkly trigger`.
```

Comment:
I think it could make sense to also mention `npx checkly trigger` here since the config also applies. The description becomes a bit long, though, and we have an awkward line-break.

![image](https://github.com/user-attachments/assets/8c986047-c50a-421c-9a2d-9bb78b9f1d48)